Client/src/Components/Options.jsx

```diff
@@ -26,10 +26,10 @@ let sample = {
   option: 'Size',
   options: [ '65"', '75"', '85"' ],
   prices: [ 5499.99,  6999.99, 10499.99],
-  insPlan2Yr: 499.99,
-  insPlan4Yr: 799.99,
-  installments: 93.74,
-  buyNow: 4999.99,
+  insPlan2Yr: [499.99, 599.99, 699.99], //changed
+  insPlan4Yr: [799.99, 1199.99, 1599.99],//changed
+  installments: [93.74, 145.83, 218.75],//changed
+  buyNow: [4999.99, 5999.99, 9499.99],// changed
   savings: 1000

```

Comment:
it doesn't break the functionality, but shouldn't this kind of info come from the database? wouldn't this being hard coded make it so that all the products have the same tags and plans?